conditionals.py

```diff
@@ -210,4 +210,22 @@ def exclude_mapcompass_info_remove(random_settings, weight_dict, **kwargs):
 def ohko_starts_with_nayrus(random_settings, weight_dict, extra_starting_items, **kwargs):
     """ If one hit ko is enabled, add Nayru's Love to the starting items """
     if random_settings['damage_multiplier'] == 'ohko':
-        extra_starting_items['starting_items'] += ['nayrus_love']
\ No newline at end of file
+        extra_starting_items['starting_items'] += ['nayrus_love']
+
+def invert_dungeons_mq_count(random_settings, weight_dict, **kwargs):
```

Comment:
Can I get you to add the conditional to `rsl_season5.json` (defaulted to off)? I like to have all available options in the current season's json so that anyone can edit that for their own custom weights easily.

Reply:
Yeah of course, with a 0 probability it should not have any impact on the current script

Reply:
done